admin_guide/allocating_node_resources.adoc

```diff
@@ -11,76 +11,69 @@
 
 toc::[]
 
-== Overview
+== Purpose for allocating node resources
 
 To provide more reliable scheduling and minimize node resource overcommitment,
-each node can reserve a portion of its resources for use by all underlying
-xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[node
-components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
-components (e.g., *sshd*, *NetworkManager*) on the host. Once specified, the
-scheduler has more information about the resources (e.g., memory, CPU) a node
-has allocated for pods.
+you can reserve a portion of the CPU and memory resources for use by the underlying xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[node components] such as kubelet, kube-proxy, and the container engine. The resources that you
+reserve are also used by the remaining system components such as *sshd*, *NetworkManager*, and so on. After you specify the resources to reserve, the scheduler has more information about the remaining memory and CPU resources that a node has available for use by pods.
 
 [[allocating-node-settings]]
-== Configuring Nodes for Allocated Resources
+== Configuring nodes for allocated resources
 
-Resources reserved for node components are based on two node settings: `kube-reserved` and `system-reserved`.
+Resources are reserved for node components and system components in {product-title} with the `system-reserved` node setting.
```

Comment:
s/with/by using ? In other words, you want to use the node setting to reserve resources. As written, "with" could be misread as "that have". (IDK, maybe they mean the same thing 😄 .)